### PR TITLE
Add ni instructions to licencing, make it more visible in quickstart

### PR DESCRIPTION
**Pull request by DimitrisJim**
This addresses #142.

Really just added the instruction of the bot to the licencing section. Tweaked some of the wording too

I also added a note to the quick-start to make sure people actually see it. If the note seems too intrusive I can just place it at the end of the 7th list item as suggested.

### CODE REVIEW

File: pullrequest.rst [hunk 1]
```diff
@@ -98,21 +98,36 @@ Licensing
...
 .. note::
    You only have to sign this document once, it will then apply to all
    your further contributions to Python.
 
-You will also need to specify you GitHub username on the
-`issue tracker`_ on the *Your Details* link found on the
-homepage.
+Here are the steps needed in order to sign the CLA:
+
+1. If you don't have an account on `bugs.python.org <issue tracker>`_,
```
Comment (brettcannon) [inline pullrequest.rst]: "(aka b.p.o)"

File: pullrequest.rst [hunk 2]
```diff
@@ -98,21 +98,36 @@ Licensing
...
+   `"Your Details" <https://cloud.githubusercontent.com/assets/2680980/23276970/d14a380c-f9d1-11e6-883d-e13b6b211239.png>`_
+   section at b.p.o.
+
+3. Fill out and sign the PSF `contributor form`_. The "bugs.python.org username"
+   requested by the form is the "Login name" field under "Your Details".
+
+After signing the CLA, please **wait at least one US business day** and
+then check "Your Details" on `b.p.o <issue tracker>`_ to see if your account has
+been marked as having signed the CLA (the delay is due to a person having
+to manually check your signed CLA).
```
Comment (brettcannon) [inline pullrequest.rst]: Probably should add a paragraph after this saying something about how to get the label to change. E.g.
"""
Once you have verified that your signing of the CLA has been noted on your b.p.o account, you can have the label updated by either asking for the CLA check to be run again, or the next time you make a code change to your PR.
"""

**Comment by brettcannon** [comment]
If we pull this in I can link to this in the bot's comment text and keep the message really short.

File: pullrequest.rst (hunk 2 above)
Comment (DimitrisJim) [inline pullrequest.rst]: I tweaked the wording a bit to make it clear that this only should be done when a PR is done and the label is added. Tell me what you think of it and if anything else needs addressing :-)

**Comment by brettcannon** [comment]
I went ahead and edited the PR directly, @DimitrisJim to tweak some wording to more directly consider the common case.

**Comment by brettcannon** [comment]
Thanks, @DimitrisJim !

**Comment by DimitrisJim** [comment]
Happy to have helped!

**Pull request merged**